shared/src/libs/packages/socket/libs/enums/room-prefixes.enum.ts

```diff
@@ -0,0 +1,6 @@
+const RoomPrefixes = {
```

Comment:
```suggestion
const RoomPrefix = {
```
don't forget about file name